src/main/java/com/diablo3CharViewer/Token.java

```diff
@@ -0,0 +1,42 @@
+package com.diablo3CharViewer;
+
+public class Token {
+
+    private String access_token;
+    private String token_type;
```

Comment:
Are the fields apart from access_token used anywhere? If not, then don't deserialize them.

Reply:
done